python/doc/examples/meta_modeling/polynomial_chaos_metamodel/plot_chaos_cleaning_strategy.py

```diff
@@ -0,0 +1,588 @@
+#
+# where :math:`p!` is the factorial number of :math:`p`.
+
+# %%
+#
+# Low-rank polynomial chaos expansion
+# -----------------------------------
+# For any :math:`\boldsymbol{\alpha} \in \mathbb{N}^p`, let :math:`\|\boldsymbol{\alpha}\|_0` be the rank of the multiindex, that is, the number of nonzero components:
+#
+#   .. math::
+#
+#     \|\boldsymbol{\alpha}\|_0 = \sum_{i = 1}^p \boldsymbol{1}_{\alpha_i > 0}
+#
+# where :math:`\boldsymbol{1}` is the indicator function.
+# The multiindex set of maximum total degree :math:`d \in \mathbb{N}` and maximum rank :math:`j \in \mathbb{N}` is ([blatman2009]_ page 74) is :
+#
+#   .. math::
+#
+#     \mathcal{A}^{d,j} = \left\{ \boldsymbol{\alpha} \in \mathbb{N}^p \; | \; \|\boldsymbol{\alpha}\|_1 \leq d, \; \; \|\boldsymbol{\alpha}\|_0 \leq j\right\}.
+#
+# Therefore, the rank-`j` polynomial chaos expansion is:
+#
+#   .. math::
+#
+#     h(\boldsymbol{\xi}) \approx \sum_{\boldsymbol{\alpha} \in \mathcal{A}^{d,j}} a_{\boldsymbol{\alpha}} \psi_{\boldsymbol{\alpha}}(\boldsymbol{\xi}).
+#
+# The rank is now an hyperparameter of the model: [blatman2009]_ suggests to use :math:`j = 2, 3, 4`. An example of low-rank PCE for the G-Sobol' function is given in [blatman2009]_ page 75.
+#
+# *Note.* It is currently not possible to create a low-rank PCE in OpenTURNS.
+
+# %%
+#
+# Model selection
+# ---------------
+# If :math:`\textrm{card}\left(\mathcal{A}^{d}\right)` is large, many coefficients may be poorly estimated, which may reduce the quality of the metamodel. We may want to select a subset of the coefficients which best predict the output. In other words, we may compute a subset
+#
+#   .. math::
+#
+#     \mathcal{A} \subseteq \mathcal{A}^{d}
+#
+# such that ([blatman2009]_ page 86) :
+#
+#   .. math::
+#
+#     h(\boldsymbol{\xi}) \approx \sum_{\boldsymbol{\alpha} \in \mathcal{A}} a_{\boldsymbol{\alpha}} \psi_{\boldsymbol{\alpha}}(\boldsymbol{\xi}).
+#
+#
+# An enumeration rule is a function from the set of integers :math:`k` to the corresponding set of multiindices :math:`\boldsymbol{\alpha}`. More precisely, let :math:`r : \mathbb{N} \rightarrow \mathbb{N}^p` be the function such that :
+#
+#   .. math::
+#
+#     r(k) = \boldsymbol{\alpha}
+#
+# for any :math:`k \geq 0`.
+# Let :math:`K \in \mathbb{N}` be a parameter representing the number of coefficients considered in the selection. Given an enumeration rule for the multiindices :math:`\boldsymbol{\alpha}`, at most :math:`K` multiindices will be considered. Let :math:`\mathcal{A}_K` be the corresponding multiindex set :
+#
+#   .. math::
+#
+#     \mathcal{A}_K = \left\{ \boldsymbol{\alpha} \; | \; r^{-1}(\boldsymbol{\alpha}) = k \leq K \right\}.
+#
+#
+# Let :math:`\epsilon > 0` be a parameter representing the minimum relative value of a significant coefficient :math:`a_{\boldsymbol{\alpha}}`. The `CleaningStrategy` uses the following criteria to select the coefficients :
```

Comment:
```suggestion
# Let :math:`\epsilon > 0` be a parameter representing the minimum relative value of a significant coefficient :math:`a_{\boldsymbol{\alpha}}`.
# The :class:`~openturns.CleaningStrategy` uses the following criterion to select the coefficients :
```
Again, link to CleaningStrategy doc + I believe "criteria" to be the plural of "criterion"?